### ISSUE TITLE
CLI Discover Functionality is Broken

### ISSUE THREAD

**Issue by pbronez** [keybase/keybase-issues#1680]
The CLI client's proof-to-key lookup is failing:

```
$ keybase id twitter://malgorithms
error: user doesn't have a public key
$ keybase id github://malgorithms
error: user doesn't have a public key
$ keybase id reddit://malgorithms
error: user doesn't have a public key
$ keybase id hackernews://malgorithms
error: user doesn't have a public key
```
Although it still handles  `id  <keybase username>`  properly:
```
$ keybase id chris
✔ public key fingerprint: 94AA 3A5B DBD4 0EA5 49CA BAF9 FBC0 7D6A 9701 6CB3
✔ "malgorithms" on twitter: https://twitter.com/malgorithms/status/433640580220874754
✔ "malgorithms" on github: https://gist.github.com/8852373
✔ "malgorithms" on reddit: https://www.reddit.com/r/KeybaseProofs/comments/2kpa4r/my_keybase_proof_redditmalgorithms_keybasechris/
✔ "bitcoyne" on coinbase: https://coinbase.com/bitcoyne/public-key
✔ "malgorithms" on hackernews: https://news.ycombinator.com/user?id=malgorithms
✔ admin of keybase.io via HTTPS: https://keybase.io/.well-known/keybase.txt
✔ admin of chriscoyne.com via HTTPS: https://chriscoyne.com/keybase.txt
✔ admin of the DNS zone for chriscoyne.com
฿ bitcoin 1MPt9BuAVM6YphzyBCNUXkh5dprThwSvbD
```

And  a direct call to the the `user/discover` API endpoint also works (output truncated):

```
$ curl https://keybase.io/_/api/1.0/user/discover.json?twitter=malgorithms
{"status": {
    "code": 0,
    "name": "OK"},
  "matches": {
    "twitter": [[{
          "username": "chris",
          "uid": "23260c2ce19420f97b58d7d95b68ca00",
          "public_key": {
            "key_fingerprint": "94aa3a5bdbd40ea549cabaf9fbc07d6a97016cb3",
            "bits": 4096,
            "algo": 1
          },
          "full_name": "Chris Coyne",
          "remote_proofs": {
            "twitter": "malgorithms",
            "github": "malgorithms",
            "reddit": "malgorithms",
            "hackernews": "malgorithms",
            "coinbase": "bitcoyne"
          }}]]}}
```

I have reproduced this with keybase users `chris`, `jeresig` and `alexisohanian`  ([who are listed in the documentation](https://keybase.io/docs/api/1.0/call/user/discover)) and myself. It appears that there's a problem with `id <proof>`. 

My environment:

```
$ keybase version
keybase (keybase.io CLI) v0.8.9
- node.js v0.10.33
- gpg (GnuPG/MacGPG2) 2.0.27
- libgcrypt 1.6.3
Identifies as: 'keybase.io node.js client v0.8.9 darwin'
```

**Comment by maxtaco** [comment]
Definitely a bug, thanks for bringing it to our attention. We'll take a look today.

**Comment by oconnor663** [comment]
Just released version 0.8.10, which should fix this issue. Please let me know if you see it again. Thanks for reporting!

**Comment by pbronez** [comment]
Upgraded to 0.8.10, everything works now. Thanks for the fast response! Excited to see where you guys take this project.
